docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,263 @@
+
+Receive XNS
+-----------
+
+To receive XNS, do the following:
+
+#. Open the dashboard of the Insolar Wallet and click the avatar icon the in upper-right corner.
+
+   .. image:: imgs/mig-test/click-avatar.png
+      :width: 250px
+
+#. In the **Your Wallet** panel, click :guilabel:`Copy XNS address`. This copies the address to the clipboard.
+
+   .. image:: imgs/mig-test/copy-xns-address.png
+      :width: 200px
+
+#. Reveal the address to anyone who wishes to transfer XNS to you and wait for the incoming transaction.
+#. View the incoming transactions: in the **Your Wallet** panel, click :guilabel:`Transaction history`.
+
+   .. image:: imgs/mig-test/click-history.png
+      :width: 200px
+
+#. On the **Transaction history** screen, open the :guilabel:`RECEIVED` tab.
+
+   .. image:: imgs/mig-test/click-received.png
+      :width: 400px
+
+Once you receive the XNS, the balance on the :guilabel:`Dashboard` tab increases.
+
+Send XNS
+--------
+
+To send XNS, do the following:
+
+#. Open the :guilabel:`Dashboard` tab in the Insolar Wallet and click :guilabel:`SEND`.
+
+   .. image:: imgs/mig-test/click-send.png
+      :width: 150px
+
+#. On the **Send XNS** screen, fill in the recipient address, amount of XNS to send, and click :guilabel:`NEXT`.
+
+   .. image:: imgs/mig-test/send-xns.png
+      :width: 400px
+
+#. Make sure your Ledger Nano is connected, unlocked, and the Insolar application is launched on it.
+#. On the **Send XNS** screen, check the following transaction details and click :guilabel:`SEND`:
+
+   * recipient address,
+   * amount of XNS to send,
+   * transaction fee,
+   * total amount — including the fee.
+
+   .. image:: imgs/mig-test/check-details.png
+      :width: 400px
+
+#. In the dashboard of the Ledger Nano device, the application prompts you to verify the transfer details and sign the :guilabel:`Send XNS` command. Click the :guilabel:`right` button to cycle through the details and check that they are the same as in the web wallet.
+  
+#. Press both :guilabel:`left` and :guilabel:`right` buttons to sign the :guilabel:`Send XNS` command.
+
+   .. image:: imgs/mig-test/ledger-s-create-account-sign.png
+      :width: 300px
+
+#. View the outgoing transactions: in the **Your Wallet** panel, click :guilabel:`Transaction history`.
+
+   .. image:: imgs/mig-test/click-history.png
+      :width: 200px
+
+#. On the **Transaction history** screen, open the :guilabel:`SENT` tab.
+
+   .. image:: imgs/mig-test/click-sent.png
+      :width: 400px
+
+Once you send XNS, the balance in the :guilabel:`Dashboard` tab decreases.
+
+Transfer swapped XNS from deposit to your main account
+------------------------------------------------------
+
+Once you've :ref:`swapped your INS into XNS <swap>` your XNS are stored in your Insolar Wallet on a deposit account. Each swap operation creates a separate deposit account that goes from the status :guilabel:`ON HOLD` to :guilabel:`RELEASED` upon a successful swap.
```

Comment:
_Missed a comma here_ : Once you've :ref:`swapped your INS into XNS <swap>`**, y**our XNS are stored in your Insolar Wallet on a deposit account.